### PR TITLE
Plugin artifact api

### PR DESCRIPTION
**Pull request by josenavas**
Creates a minimal API that the plugins will use to request artifact information from Qiita.
It is build on top of #1550 so review/merge that one first to remove those commits from here.

### CODE REVIEW

**Comment by antgonza** [comment]
:+1:

File: qiita_db/handlers/artifact.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+class ArtifactFilepathsHandler(RequestHandler):
+    def get(self, artifact_id):
+        """Retrieves the filepath information of the given artifact
+
+        Parameters
+        ----------
+        artifact_id : str
+            the id of the artifact that the information needs to be returned
```
Comment (mortonjt) [inline qiita_db/handlers/artifact.py]: This is a bit confusing to read. Could you reword this to something like

`the id of the artifact whose file path information is being retrieved`.

**Comment by mortonjt** [comment]
One nit-picky comment.  Otherwise, I think it looks good.
